homeassistant/components/otp/sensor.py

```diff
@@ -69,11 +79,11 @@ def _call_loop(self):
         self.hass.loop.call_later(self._next_expiration, self._call_loop)
 
     @property
-    def name(self):
+    def name(self) -> str:
```

Comment:
Preliminary PR => can this be moved to `attr_name`?

Reply:
New PR as suggested
* https://github.com/home-assistant/core/pull/118511